app/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,7 @@
     <string name="two_factors_otp_error">Two factors OTP is required</string>
     <string name="no_issue">No issue</string>
     <string name="copy">Copy</string>
+    <string name="copy_repo_url">Copy repo URL</string>
```

Comment:
why would you specifically mention repo URL? I guess by now users know that copy means copy URL.